vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -17279,10 +17279,71 @@ unit:QT
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
   rdfs:label "Quart" ;
 .
+unit:QT_UK
+  a qudt:Unit ;
+  qudt:conversionMultiplier "0.0011365225"^^xsd:double ;
+  qudt:conversionOffset 0e+00 ;
+  qudt:description "unit of the volume for fluids according to the Imperial system of units" ;
```

Comment:
descriptions that are plain text should use the property qudt:plainTextDescription. qudt:description is being deprecated. If the description is likely to carry LaTeX it should be a dct:description either with the literal type qudt:LatexString.